ush/ufoeval/run_ufo_hofx_test.sh

```diff
@@ -10,21 +10,57 @@
 # - Run EVA
 #-------------------------------------------------------------
 #--------------- User modified options below -----------------
-cycle=2021080100
-obtype=amsua_n19
-GDASApp=/work2/noaa/da/cmartin/GDASApp/dev/GDASApp # Change this to your own branch
-workdir=/work2/noaa/da/$LOGNAME/ufoeval/$cycle/$obtype
-yamlpath=$GDASApp/parm/atm/obs/testing/amsua_n19.yaml
+obtype=$1
+cycle=${2:-2021080100}
+
+if [ $# -lt 1 ] || [ $# -gt 2 ]; then
+   echo "Incorrect number of arguments"
+   echo "Usage:"
+   echo $0 instrument [cycle]
+   exit 1
+fi
+
+machine=hera
+#GDASApp=/work2/noaa/da/cmartin/GDASApp/dev/GDASApp # Change this to your own branch
+GDASApp=/scratch1/NCEPDEV/da/Andrew.Collard/git/GDASApp_sprint-ioda-converters/ # Change this to your own branch
+
+if [ $machine = orion ]; then
+   workdir=/work2/noaa/da/$LOGNAME/ufoeval/$cycle/$obtype
+elif [ $machine = hera ]; then
+   workdir=/scratch1/NCEPDEV/stmp2/$LOGNAME/ufoeval/$cycle/$obtype
+else
+   echo "Machine " $machine "not found"
+   exit
+fi
+
+yamlpath=$GDASApp/parm/atm/obs/testing/${obtype}.yaml
 exename=test_ObsFilters.x
-machine=orion
-radiance="YES"
 
 #-------------- Do not modify below this line ----------------
 # paths that should only be changed by an expert user
-GeoDir=/work2/noaa/da/cmartin/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20220816/geovals/
-ObsDir=/work2/noaa/da/cmartin/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20220816/obs/
-BCDir=/work2/noaa/da/cmartin/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20220816/bc/
-FixDir=/work2/noaa/da/cmartin/GDASApp/fix/
+
+obtype_short=${obtype:0:4}
+if [ $obtype_short = "cris" ] || [ $obtype_short = "iasi" ] || [ $obtype_short = "hirs" ] || [ $obtype_short = "sevi" ] || \
+   [ $obtype_short = "avhr" ] || [ $obtype_short = "mhs_" ] || [ $obtype_short = "ssmi" ]; then
+   radiance="YES"
+else 
+   radiance="NO"
+fi
+
+if [ $machine = orion ]; then
+    export Datapath='/work2/noaa/da/cmartin/UFO_eval/data/gsi_geovals_l127/nofgat_aug2021/20220816' 
```

Comment:
Use the new date on this one too, and also perhaps something like `$dataprocdate` that can be used in both of these?

Reply:
Done